--- a/meson.build       2024-03-17 10:22:05.830646816 +0000
+++ b/meson.build       2024-03-17 10:21:34.090646854 +0000
@@ -706,7 +706,7 @@
 ############################################################

 python = find_program('python3')
-if run_command(python, '-c', 'import jinja2', check : false).returncode() != 0
+if run_command(python, '-c', '"import jinja2"', check : false).returncode() != 0
         error('python3 jinja2 missing')
 endif
 